symfony/messenger/4.1/config/packages/messenger.yaml

```diff
@@ -0,0 +1,9 @@
+framework:
+    messenger:
+        adapters:
+            # Uncomment the following line to enable an adapter named "amqp"
+            # amqp: "%env(MESSENGER_ADAPTER_DSN)%"
+
+        routing:
+            # Route your messages to the senders...
+            # 'App\Message\YourMessage': messenger.sender.amqp
```

Comment:
We're discussing allowing the short adapter name here - e.g. `amqp` (and also service id's, of course). If we do that (I want to), it might be simpler to do that before merging the recipe.

Reply:
Fair point. For the records, it's here: https://github.com/symfony/symfony/pull/26909